scripts/darken_colors.lua

```diff
@@ -0,0 +1,164 @@
+-- Plugin lifepillar/vim-colortemplate must be available on &runtimepath
```

Comment:
Let's rename this to `generate_colours.lua`.

I like having everything in one place.